src/System.Windows.Forms/src/System/Windows/Forms/DataGridView/DataGridView.Methods.cs

```diff
@@ -17008,19 +16561,19 @@ protected override void OnMouseMove(MouseEventArgs e)
             CursorInternal = Cursors.SizeWE;
             return;
         }
-        else if (_dataGridViewOper[OperationTrackRowResize] ||
-            _dataGridViewOper[OperationTrackColHeadersResize] ||
-            ((hti._typeInternal == DataGridViewHitTestTypeInternal.RowResizeBottom ||
-            hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeTop ||
-            hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeBottom ||
-            hti._typeInternal == DataGridViewHitTestTypeInternal.RowResizeTop ||
-            hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeTop ||
-            hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeBottom) &&
-            !_dataGridViewOper[OperationTrackRowHeadersResize] &&
-            !_dataGridViewOper[OperationTrackColRelocation] &&
-            !_dataGridViewOper[OperationTrackColSelect] &&
-            !_dataGridViewOper[OperationTrackRowSelect] &&
-            !_dataGridViewOper[OperationTrackCellSelect]))
+        else if (_dataGridViewOper[OperationTrackRowResize]
+            || _dataGridViewOper[OperationTrackColHeadersResize]
+            || ((hti._typeInternal == DataGridViewHitTestTypeInternal.RowResizeBottom
+                    || hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeTop
+                    || hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeBottom
+                    || hti._typeInternal == DataGridViewHitTestTypeInternal.RowResizeTop
+                    || hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeTop
+                    || hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeBottom)
```

Comment:
This seems like it has an extra level of indentation but maybe that was intentional
```suggestion
                || hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeTop
                || hti._typeInternal == DataGridViewHitTestTypeInternal.TopLeftHeaderResizeBottom
                || hti._typeInternal == DataGridViewHitTestTypeInternal.RowResizeTop
                || hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeTop
                || hti._typeInternal == DataGridViewHitTestTypeInternal.ColumnHeadersResizeBottom)
```